website/docs/r/spot_fleet_request.html.markdown

```diff
@@ -78,6 +78,91 @@ resource "aws_spot_fleet_request" "foo" {
 }
 ```
 
+### Example with a dynamic block
+
+```hcl
+resource "aws_spot_fleet_request" "cheap_compute" {
+  iam_fleet_role                      = "${aws_iam_role.aws-ec2-spot-fleet-tagging-role.arn}"
+  target_capacity                     = var.target_capacity
+  valid_until                         = "${timeadd(timestamp(), var.timeadd)}"
+  allocation_strategy                 = var.allocation_strategy
```

Comment:
We usually prefer to show constant values or expressions transforming constant values in these resource examples, so the reader can see what sort of value is expected. Could these variable references be replaced with constant values, similar to the other examples on this page?

Reply:
yeah I can easily replace most vars with dummy values